bin/ebuild

```diff
@@ -129,13 +128,16 @@ try:
     if not opts.color == "y" and (
         opts.color == "n"
         or portage.settings.get("NOCOLOR") in ("yes", "true")
```

Comment:
I think I'd prefer this refactored into a function that we can call in other places too.

Reply:
Yes that would be nice I'll do that